### ISSUE TITLE
Genus Myzinum is currently in family Tiphiidae but needs to be moved to be in family Thynnidae (https://www.gbif.org/species/4407017)

### ISSUE THREAD

**Issue by bdagley** [gbif/backbone-feedback#145]
https://www.gbif.org/species/4407017

**Comment by jhnwllr** [comment]
// json for auto-checking
{
"name": "Myzinum Latreille, 1803",
"wrongGroup": "Tiphiidae",
"rightGroup": "Thynnidae"
}
[why is this here?](https://github.com/gbif/backbone-feedback/wiki/JSON-comments-for-automation-%E2%80%90-Experimental)

**Comment by camiplata** [comment]
For the xrelease we take the genera from [TAXREF wich uses Tiphiidae](https://www.checklistbank.org/dataset/2008/taxon/791834), additional species come from a [zootaxa paper (2009) that also uses Tiphiidae](https://www.mapress.com/zt/article/view/zootaxa.2224.1.2). [Bold and iNaturalist use Thynnidae](http://bins.boldsystems.org/index.php/TaxBrowser_Taxonpage?taxid=174433). 

COL currently lacks an hymenoptera expert and we don't have a source for this group. For now we will address the issue on the extended release options:

1. Ask TAXREF to update its lists
2. Find a better source for this group, @bdagley any suggestion?

**Comment by bdagley** [comment]
Hi @camiplata, 
re: sources, I believe that the taxonomy on BugGuide, iNaturalist, and ITIS are currently correct, or represent the prevailing taxonomic view. I tried loading ITIS but it timed out. I don't currently recall some of the earlier sources; it was other users who originally pointed this out to me. I did find one older article that may support the placement in Thynnidae, [Pilgrim et al. (2008)](https://www.researchgate.net/profile/Carol-Von-Dohlen/publication/229792867_Molecular_phylogenetics_of_Vespoidea_indicate_paraphyly_of_the_superfamily_and_novel_relationships_of_its_component_families_and_subfamilies/links/6405ec87b1704f343faa9c2c/Molecular-phylogenetics-of-Vespoidea-indicate-paraphyly-of-the-superfamily-and-novel-relationships-of-its-component-families-and-subfamilies.pdf), although I assume there are also more recent articles. The 2009 zootaxa paper possibly was before the revision became more of a prevailing view.

**Comment by camiplata** [comment]
Thanks @bdagley! 
Actually ITIS has no coverage of neither Myzinum or Thynnidae

I understand you are a curator of iNat Taxonomy (right?), Is there any other Hymenoptera group that you are actively curating where you think the iNat taxonomy could be a well scrutinized source?

**Comment by bdagley** [comment]
Yes, I and other identifiers, most of them taxonomists or graduate students, have curated the iNat and BugGuide taxonomies. Vespidae, for example, is very accurate on those websites (which was discussed in some COL, EOL, or GBIF-related GitHub Issues I created in the past; there are currently problems with the Vespidae taxonomy on GBIF, but it was due to an different organization sending incorrect information). Bees are also well-curated, with the caveat that there are multiple interpretations in the current be literature over what taxonomic classification should be used for some groups of bees. Among the aculeate wasps, I'm most familiar with Vespidae, which is why I may not have all of the sources for the Thynnidae issue on hand or in memory at the moment. I believe taxonomists such as the following (https://www.inaturalist.org/people/matthias22), or others who worked more directly with Thynnidae taxonomy on those websites, could provide further details or sources if someone were to contact them.

**Comment by camiplata** [comment]
I'll be testing the iNat Vespoidea in the xrelease

**Comment by bdagley** [comment]
@camiplata Thanks very much.

**Comment by jhnwllr** [comment]
## 🤖 AUTOMATED VALIDATION REPORT





**Issue Type:** `wrongGroup ` | **Status:** ❌ OPEN

**Name:** Myzinum Latreille, 1803  
**Wrong Group:** Tiphiidae Leach, 1915  
**Right Group:** Thynnidae Schuckard, 1840

**Found in COL:** [Myzinum Latreille, 1803](https://www.checklistbank.org/dataset/3LXR/taxon/QK6CF) (QK6CF), [Myzinum Latreille, 1803](https://www.checklistbank.org/dataset/3LXR/taxon/VMGF5) (VMGF5)

**[ID QK6CF](https://www.checklistbank.org/dataset/3LXR/taxon/QK6CF):**
- Classification: Tiphiidae Leach, 1915 > Vespoidea > Aculeata > Apocrita > Hymenoptera > Insecta > Hexapoda > Arthropoda > Animalia > Eukaryota (Chatton, 1925) Whittaker & Margulis, 1978
- Contains wrong group (Tiphiidae Leach, 1915): ❌ Yes
- Contains right group (Thynnidae Schuckard, 1840): ❌ No

**[ID VMGF5](https://www.checklistbank.org/dataset/3LXR/taxon/VMGF5):**
- Classification: Animalia > Eukaryota (Chatton, 1925) Whittaker & Margulis, 1978
- Contains wrong group (Tiphiidae Leach, 1915): ✅ No
- Contains right group (Thynnidae Schuckard, 1840): ❌ No

---

<sub>🔄 This comment is automatically updated when the validation status changes. Last updated: 2026-07-08T13:21:05Z</sub>
